config.py

```diff
@@ -13,116 +15,125 @@
 from calibre.utils.config import JSONConfig
 from calibre.gui2 import error_dialog
 
-prefs = JSONConfig('plugins/xray_creator')
+__prefs__ = JSONConfig('plugins/xray_creator')
```

Comment:
Why did pylint want this change?  Given we then import it elsewhere (and rename as prefs, instead of __prefs__) this seems odd.  But possibly some PEP rule doesn't like this naming of a global variable..